client/src/components/atoms/DateOfBirthPicker/DateOfBirthPicker.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+const DateOfBirthPicker = () => {
+  const [startDate, setStartDate] = useState(null);
+  return (
+    <DatePicker
+      selected={startDate}
```

Comment:
startDate null ?

Reply:
If I put the current date here. It displays instead of the placeholder.